app/server/src/controllers/odinController.ts

```diff
@@ -3,10 +3,12 @@ import { APIService } from "../apiService";
 
 export class OdinController {
     static getRunner = async (req: Request, res: Response) => {
-        await APIService.get("/support/runner-ode", req, res);
+        await new APIService(req, res)
+            .get("/support/runner-ode");
```

Comment:
is there any reason you don't want to export initialised instance? `export const api = () => new APIService();`, in my opinion it makes cleaner usage 

`await api(req, res).get("/support/runner-ode");`

instead of having new APIService every time.

Reply:
Sorry, I think I misinterpreted your original suggestion. Yep, this is terser.